firstvoices/backend/views/custom_order_recalculate_views.py

```diff
@@ -67,21 +68,26 @@ def create(self, request, *args, **kwargs):
 
         # Call the recalculation task
         try:
-            recalculation_results = recalculate_custom_order.apply_async((site_slug,))
-
-            # Delete any previous results
-            CustomOrderRecalculationResult.objects.filter(
+            # Check if preview task for the same site is ongoing
```

Comment:
super minor: this comment is about the preview, but this code is for the real recalculation.